src/routes/api/index.js

```diff
@@ -11,6 +12,7 @@ usersRouter.use('/v1/users', passwordRouter);
 usersRouter.get('/', (req, res) => res.status(200).send('Welcome to  Shadowcat API'));
 
 usersRouter.use('/v1/users', passwordRouter);
+usersRouter.use('/v1/trips', returntrip);
```

Comment:
All trip requests for oneway, return and multi-city should use just one route based on the advise of @Stylll